doc/requirements.txt

```diff
@@ -32,7 +32,7 @@ torch==1.9.0; sys_platform == "darwin" and python_version < "3.10"
 torch==1.13.1; sys_platform != "darwin" and python_version == "3.10"
 torch==1.13.1; sys_platform == "darwin" and python_version == "3.10"
 jinja2==3.0.3
-rustworkx==0.12.1
+rustworkx==0.15.1
```

Comment:
We may want to consider adding a pin in `setup.py`, if we have a hard lower bound.

Is there a way we could accomodate more versions of rustworkx?

Reply:
According to their release notes, the strategy argument was added on 0.15, so that would be our lower bound: https://www.rustworkx.org/dev/release_notes.html#relnotes-0-15-0

Reply:
Do I then add to the setup.py file:

```
networkx>=0.15
```
?

Reply:
From discussion in person: decided to add compatibility with at least 0.14 by checking if the `strategy` attribute is available in the colouring function from `rustworkx`.